.vscode/settings.json

```diff
@@ -8,5 +8,14 @@
     "yaml": true
   },
   "github.copilot.editor.enableCodeActions": true,
-  "github.copilot.editor.enableAutoCompletions": true
+  "github.copilot.editor.enableAutoCompletions": true,
+  "cSpell.words": [
+    "yumemi",
```

Comment:
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey) 
Added cSpell.words to make it easier to spot typos.